airflow/jobs.py

```diff
@@ -1066,6 +1066,8 @@ def _execute(self):
             job_id=self.id,
             pool=self.pool,
         )
+        # TODO alter the command to impersonate (sudo su)
+        # note that `self.task_instance.run_as_user` is in scope here
         self.process = subprocess.Popen(['bash', '-c', command])
```

Comment:
Probably want to remove the bash call: call airflow directly, it reduces security risk.